### PR TITLE
Use OPENSSL_zalloc to avoid uninitialized read

### PR DESCRIPTION
**Pull request by bonniegong**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: ssl/pqueue.c [hunk 1]
```diff
@@ -17,7 +17,7 @@ struct pqueue_st {
 
 pitem *pitem_new(unsigned char *prio64be, void *data)
 {
-    pitem *item = OPENSSL_malloc(sizeof(*item));
+    pitem *item = OPENSSL_zalloc(sizeof(*item));
```
Comment (paulidale) [inline ssl/pqueue.c:20]: Why is this necessary?
Could you please clarify what uninitialised read is occurring?

**Comment by t8m** [comment]
I am closing this as we are fully initializing the members in the function.
